src/bin/nps.js

```diff
@@ -102,6 +102,20 @@ function onInit() {
   ))
 }
 
+function getConfigType() {
+  let configType
```

Comment:
I was also not really satisfied with how it looked, so I think the ternary's the better way to go :)